### PR TITLE
List requirements

### PR DESCRIPTION
**Pull request by tsauerwein**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 UI Application for camptocamp.org v6
 ====================================
 
+Requirements
+------------
+
+ * Python 2.7
+ * Node (4.0 or higher)
```
Comment (gberaudo) [inline README.md]: Why do we need a node version so new?
I have been using v0.10.25 (from Ubuntu 15.10) without any issue.

Comment (tsauerwein) [inline README.md]: Version v0.10.29 is the one we are having problems with. I know that it works with Node 4.0. But we can take a lower version if you tell me which one is ok. :)

Comment (mfournier) [inline README.md]: I confirm node v4 isn't required, from what I've seen. I only needed v4 to get a slightly more descriptive error message which allowed me to undertand node was attempting to run java behind the scene.

Comment (tsauerwein) [inline README.md]: Ok, I changed it to:

> Node (0.10.x or higher, there are known issues with 0.10.29)

**Comment by asaunier** [comment]
:+1:

**Pull request merged**